packages/tf-next/src/commands/list-deployments.ts

```diff
@@ -1,23 +1,53 @@
-import { ApiGatewayV2 } from 'aws-sdk';
+import { DynamoDB } from 'aws-sdk';
 
-const apiGatewayV2 = new ApiGatewayV2();
+const jp = require('jsonpath');
+
+const dynamoDB = new DynamoDB();
```

Comment:
I am setting the environment variables before running the commands, because our users need to be MFA-authenticated. I also currently run with `AWS_REGION=us-east-1 ./index.js list-deployments`. I am not sure if more people have `AWS_DEFAULT_REGION` set than `AWS_REGION`. The developer experience could definitely be better. I am not quite sure yet what to do about it exactly.

Reply:
Thanks for explaining. I used `AWS_REGION` to confirm that it works.